assets/styl/_settings.styl

```diff
@@ -3,52 +3,61 @@
  * - Define generic|extendable styles
  */
 /* Grid */
-$columns = 12 // number of columns
-$gutter = 12 // guuter width
+$columns ?= 12 // number of columns
+$gutter ?= 12 // gutter width
+/* Default primary grid breakpoints */
+$breakpoint ?= {
+    'sm': "screen and (min-width: 420px)"
+    'md': "screen and (min-width: 768px)"
+    'lg': "screen and (min-width: 960px)"
+}
+/* Secondary breakpoints */
+$tweakpoint ?= {}
+$grid ?= merge($breakpoint, $tweakpoint)
```

Comment:
`$grid` is not relative to the grid system, it's a collection off breakpoint for mediaqueries

this variable name does not seem appropriate.